services/mailgun.js

```diff
@@ -0,0 +1,42 @@
+const Mailgun = require('mailgun-js');
+require('dotenv').config()
+
+
+//Your api key, from Mailgun’s Control Panel
+const api_key = process.env.API_KEY
+
+//Your domain, from the Mailgun Control Panel
+
+const domain = process.env.DOMAIN
+
+//Your sending email address
+const from_who = process.env.FROM_WHO;
+
+const mailgunModule = {}
+
+
+mailgunModule.sendConfirmationEmail = (toEmail) => {
+
+    //Settings
+    
+    let mailgun = new Mailgun({apiKey: api_key, domain: domain});
```

Comment:
`const`

Reply:
move this outside the function. You don't want to reinitialize this object everytime the function is called.